src/data/api/datatyper/personalia.tsx

```diff
@@ -0,0 +1,169 @@
+	postnummer: StringOrNothing;
+	husnummer: StringOrNothing;
+	husbokstav: StringOrNothing;
+	kommunenummer: StringOrNothing;
+	kommune: StringOrNothing;
+	adressenavn: StringOrNothing;
+	tilleggsnavn: StringOrNothing;
+	poststed: StringOrNothing;
+}
+
+export interface Matrikkeladresse {
+	matrikkelId: OrNothing<number>;
+	bruksenhetsnummer: StringOrNothing;
+	tilleggsnavn: StringOrNothing;
+	postnummer: StringOrNothing;
+	kommunenummer: StringOrNothing;
+	kommune: StringOrNothing;
+	poststed: StringOrNothing;
+}
+
+export interface Utenlandskadresse {
+	adressenavnNummer: StringOrNothing;
+	bygningEtasjeLeilighet: StringOrNothing;
+	postboksNummerNavn: StringOrNothing;
+	postkode: StringOrNothing;
+	bySted: StringOrNothing;
+	regionDistriktOmraade: StringOrNothing;
+	landkode: StringOrNothing;
+}
+
+export interface Ukjentbosted {
+	bostedskommune: StringOrNothing;
+	kommune: StringOrNothing;
+}
+
+export interface Postboksadresse {
+	postbokseier: StringOrNothing;
+	postboks: StringOrNothing;
+	postnummer: StringOrNothing;
+	poststed: StringOrNothing;
+}
+
+export interface PostadresseIFrittFormat {
+	adresselinje1: StringOrNothing;
+	adresselinje2: StringOrNothing;
+	adresselinje3: StringOrNothing;
+	postnummer: StringOrNothing;
+	poststed: StringOrNothing;
+}
+
+export interface UtenlandskadresseIFrittFormat {
+	adresselinje1: StringOrNothing;
+	adresselinje2: StringOrNothing;
+	adresselinje3: StringOrNothing;
+	byEllerStedsnavn: StringOrNothing;
+	postkode: StringOrNothing;
+	landkode: StringOrNothing;
+}
+
+export interface Bostedsadresse {
+	coAdressenavn: StringOrNothing;
+	vegadresse: OrNothing<Vegadresse>;
+	matrikkeladresse: OrNothing<Matrikkeladresse>;
+	utenlandskAdresse: OrNothing<Utenlandskadresse>;
+	ukjentBosted: OrNothing<Ukjentbosted>;
+}
+
+export interface Oppholdsadresse {
+	coAdressenavn: StringOrNothing;
+	vegadresse: OrNothing<Vegadresse>;
+	matrikkeladresse: OrNothing<Matrikkeladresse>;
+	utenlandskAdresse: OrNothing<Utenlandskadresse>;
+}
+
+export interface Kontaktadresse {
+	type: String;
+	coAdressenavn: StringOrNothing;
+	vegadresse: OrNothing<Vegadresse>;
+	postboksadresse: OrNothing<Postboksadresse>;
+	utenlandskAdresse: OrNothing<Utenlandskadresse>;
+	postadresseIFrittFormat: OrNothing<PostadresseIFrittFormat>;
+	utenlandskAdresseIFrittFormat: OrNothing<UtenlandskadresseIFrittFormat>;
+}
+
+export interface PersonaliaEpost {
+	epostAdresse: StringOrNothing;
+	epostSistOppdatert: StringOrNothing;
+	master: StringOrNothing;
+}
+
+export interface PersonaliaV2Info extends GrunnPersonalia {
+	barn: PersonsBarn[];
+	kontonummer: string;
+	geografiskEnhet: OrNothing<Enhet>;
+	telefon: PersonaliaTelefon[];
+	epost: OrNothing<PersonaliaEpost>;
+	statsborgerskap: string[];
+	partner?: PersonaliaPartner;
+	sivilstandliste?: PersonaliaSivilstandNy[];
+	bostedsadresse: OrNothing<Bostedsadresse>;
+	oppholdsadresse: OrNothing<Oppholdsadresse>;
+	kontaktadresser: Kontaktadresse[];
+	malform: StringOrNothing;
```

Comment:
Det hentes inn veldig mye data, er planen å hente inn "alt" og så heller fjerne det dersom det ikke brukes? :)

Reply:
Tror planer er å se igjennom etter oppsettet med backend er ferdig, også bestemme hvilken approach vi skal gå for deretter!